src/testArrays/ArrayCopy.java

```diff
@@ -0,0 +1,24 @@
+package testArrays;
+
+import java.util.Arrays;
+
+public class ArrayCopy {
+    public static void main(String[] args) {
+        int[] first = {1, 5, 23, 11};
+        int[] second = {3, 12, 7, 8, 10, -4};
+        arraycopy(first, 1, second, 2, 2);
+        System.out.println(Arrays.toString(second));
+    }
+
+    private static void arraycopy(int[] arr1, int sourcePos, int[] arr2, int destPos, int length) {
```

Comment:
`arr1` and `arr2` is very bad naming. I propose `src` and `dst` or `source` and `destination`.